examples/multilang/mycomponent/python/__init__.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import os
+from pulumi import ComponentResource, Output,  ResourceOptions, Input, Inputs
+from pulumi.remote import ProxyComponentResource
+from pulumi.runtime import register_proxy_constructor
+from typing import Callable, Any, Dict, List, Optional
+from pulumi_aws import ec2
+
+class MyInnerComponent(ProxyComponentResource):
+    data: Output[str]
+    def __init__(__self__, resource_name: str, opts: Optional[ResourceOptions]=None) -> None:
+        super().__init__(
+            "my:mod:MyInnerComponent",
+            resource_name,
+            os.path.abspath(os.path.join(os.path.dirname(__file__),"..")),
```

Comment:
same question about base type figuring this out.  if not possible, might be good to have a helper for this as it's super ugly.

also, this will likely need to ba  relative path not an absolute path in practice in a real solution right?

Reply:
I think the general strategy here *is* what we want:
1. The proxy code computes paths relative to itself (since it was code generated by someone who put the library somewhere relative to it)
2. The proxy code turns those into absolute paths to pass across the RPC boundary to another language runtime
3. That language runtime uses those paths to load the module.

This seems like the least fragile approach here.  No doubt helpers to compute these things will make sense - but this is also code that will eventually be auto-generated, and is dependent on how the author chooses to layout packages - so I expect leaving this explicit for now will be okay.